### PR TITLE
Provide virtio RNG by default by changing QEMU_VIRTIO_RNG default to 1

### PR DESCRIPTION
**Pull request by asdil12**
https://bugzilla.suse.com/show_bug.cgi?id=1197595

### CODE REVIEW

**Comment by mloviska** [comment]
LGTM!

**Comment by Vogtinator** [comment]
Is there any hardware which might not support it?

File: backend/qemu.pm [hunk 1]
```diff
@@ -818,6 +818,7 @@ sub start_qemu ($self) {
         }
 
         # Keep additional virtio _after_ Ethernet setup to keep virtio-net as eth0
+        $vars->{QEMU_VIRTIO_RNG} //= 1;
         if ($vars->{QEMU_VIRTIO_RNG}) {
```
Comment (Martchus) [inline backend/qemu.pm:822]: I'd prefer
```suggestion
        if ($vars->{QEMU_VIRTIO_RNG} //= 1) {
```
or even just
```suggestion
        if ($vars->{QEMU_VIRTIO_RNG} // 1) {
```

**Review by Martchus — COMMENTED** [review]
I suppose this LGTM. The only requirement for this to work is `/dev/urandom` being present on the worker host, right? Then it should work on all worker hosts.

**Review by Martchus — APPROVED** [review]

**Comment by asdil12** [comment]
Can we merge this?

**Review by okurz — APPROVED** [review]

**Pull request merged**